source/manual/tech-lead-responsibilities.html.md

```diff
@@ -28,8 +28,6 @@ Anybody on the team should feel comfortable picking up any card, even if they do
 
 For existing teams, tech leads should on-board new developers to the team. You should meet with developers new to the team to introduce the team’s work and overall goals. Discuss the new developer’s objectives so you can help ensure there is suitable work for them and generally help settle them in to be productive team members.
 
-Ensure the developer is in the correct team in the [seal][].
```

Comment:
It should be safe to remove the associated link reference now, too:

https://github.com/alphagov/govuk-developer-docs/blob/3cb981ba047551fbf4481ba1b5ae951a97015b89/source/manual/tech-lead-responsibilities.html.md?plain=1#L33